aws/ec2/helpers.py

```diff
@@ -293,6 +293,30 @@ def is_ebs_volume_encrypted(ebs):
     return ebs["Encrypted"]
 
 
+def is_ebs_volume_piops(ebs):
+    """
+    Checks if the EBS volume type is provisioned iops
+
+    >>> is_ebs_volume_piops({'VolumeType': 'io1'})
+    True
+    >>> is_ebs_volume_piops({'VolumeType': 'standard'})
+    False
+    >>> is_ebs_volume_piops({})
+    Traceback (most recent call last):
+    ...
+    KeyError: 'VolumeType'
+    >>> is_ebs_volume_piops(0)
+    Traceback (most recent call last):
+    ...
+    TypeError: 'int' object is not subscriptable
+    >>> is_ebs_volume_piops(None)
+    Traceback (most recent call last):
+    ...
+    TypeError: 'NoneType' object is not subscriptable
```

Comment:
Note to self: if we start typehinting e.g. with typeddict we can reduce the number of these types to unit test